tests/e2e/specs/swap-tokens.spec.js

```diff
@@ -3,48 +3,104 @@ describe('Swap Tokens Tests', () => {
   const limitFloat = float => parseFloat(float.toFixed(5));
   const amountToSwap = 0.001;
   const transactionFee = 0.2;
+  const walletAddress = {
+    value: null,
+  };
   const phrasesList = {
     emerynet: {
       walletButton: 'li[data-value="testnet"]',
       psmNetwork: 'Agoric Emerynet',
       token: 'ToyUSD',
+      isLocal: false,
+      provisionFee: 0.75,
     },
     local: {
       walletButton: 'li[data-value="local"]',
       psmNetwork: 'Local Network',
       token: 'USDC_axl',
+      isLocal: true,
+      provisionFee: 0,
     },
   };
   const networkPhrases = phrasesList[Cypress.env('AGORIC_NET')];
 
   it(`should connect with Agoric Chain on https;//wallet.agoric.app`, () => {
-    cy.origin('https://wallet.agoric.app/', () => {
-      cy.visit('/');
-    });
-    cy.acceptAccess().then(taskCompleted => {
-      expect(taskCompleted).to.be.true;
-    });
-
-    cy.origin(
-      'https://wallet.agoric.app/',
-      { args: { networkPhrases } },
-      ({ networkPhrases }) => {
+    if (networkPhrases.isLocal) {
+      cy.setupWallet();
+
+      cy.origin('https://wallet.agoric.app/', () => {
```

Comment:
Just created https://github.com/Agoric/dapp-psm/pull/142 so that we can do `suggestChain` without visiting the wallet app, would like to remove this after that PR goes in.

Reply:
sounds good

Reply:
Just merged #142

Reply:
removed https://wallet.agoric.app/ calls from local tests